frontend/index.html

```diff
@@ -7,22 +7,36 @@
 		<link
 			rel="apple-touch-icon"
 			sizes="180x180"
-			href="/apple-touch-icon.png"
+			href="./src/apple-touch-icon.png"
 		/>
+		<link rel="apple-touch-icon" href="./src/logo192.png" />
 		<link
 			rel="icon"
 			type="image/png"
 			sizes="32x32"
-			href="/favicon-32x32.png"
+			href="./src/favicon-32x32.png"
 		/>
 		<link
 			rel="icon"
 			type="image/png"
 			sizes="16x16"
-			href="/favicon-16x16.png"
+			href="./src/favicon-16x16.png"
+		/>
+		<link
+			rel="mask-icon"
+			href="./src/safari-pinned-tab.svg"
+			color="#5629c6"
+		/>
+		<!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+		<link rel="manifest" href="/manifest.json" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap"
+			rel="stylesheet"
 		/>
-		<link rel="manifest" href="/site.webmanifest" />
```

Comment:
Removed this webmanifest because it seems to be mostly empty, and there's another manifest at manifest.json that's referred to later on and actually populated.